Debian/8/Makefile

```diff
@@ -127,5 +127,5 @@ dist: guide content
 clean:
 	rm -f $(OUT)/*.xml $(OUT)/*.html $(OUT)/*.xhtml $(OUT)/*.pdf  $(OUT)/*.spec $(OUT)/*.tar $(OUT)/*.gz $(OUT)/*.ini $(OUT)/*.csv
 	rm -rf $(DIST)/content $(DIST)/guide
-	$(SHARED)/utils/build-remediations.py --output ./templates clean
+	rm -rf ./templates/output
```

Comment:
Ah, right. I misunderstood you. I definitely agree, I am just not sure if I want to be doing that many changes in one PR. I want to avoid creating conflicts with @ybznek 's work on cleaning up the template duplication in various SSG products.